src/proto_bcd/findings/suggested_commits.py

```diff
@@ -0,0 +1,186 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from collections import defaultdict
+from types import MappingProxyType
+from proto_bcd.findings.finding_category import FindingCategory
+
+_templates = defaultdict(lambda: "Unknown change type")
+_templates[FindingCategory.ENUM_VALUE_ADDITION] = (
+    "add enum value `{context}.{subject}`."
+)
+_templates[FindingCategory.ENUM_VALUE_REMOVAL] = (
+    "remove enum value `{context}.{subject}`."
+)
+_templates[FindingCategory.ENUM_VALUE_NAME_CHANGE] = (
+    "rename enum value `{context}.{oldsubject}` to `{context}.{subject}`."
+)
+_templates[FindingCategory.ENUM_VALUE_NUMBER_CHANGE] = (
+    "change enum value `{context}.{oldsubject}` to `{context}.{subject}`."
+)
+_templates[FindingCategory.ENUM_ADDITION] = "add enum `{subject}`."
+_templates[FindingCategory.ENUM_REMOVAL] = "remove enum `{subject}`."
+_templates[FindingCategory.FIELD_ADDITION] = "add field `{context}.{subject}`."
+_templates[FindingCategory.FIELD_REMOVAL] = "remove field `{context}.{subject}`."
+_templates[FindingCategory.FIELD_NAME_CHANGE] = (
+    "rename field `{context}.{oldsubject}` to `{context}.{subject}`."
+)
+_templates[FindingCategory.FIELD_REPEATED_CHANGE] = (
+    "change repeated flag of field `{context}.{subject}`."
+)
+_templates[FindingCategory.FIELD_TYPE_CHANGE] = (
+    "change type of field `{context}.{subject}` from `{oldtype}` to `{type}`."
+)
+_templates[FindingCategory.FIELD_ONEOF_MOVE_OUT] = (
+    "move field `{context}.{subject}` out of oneof."
+)
+_templates[FindingCategory.FIELD_ONEOF_MOVE_IN] = (
+    "move field `{context}.{subject}` into oneof."
+)
+_templates[FindingCategory.FIELD_PROTO3_OPTIONAL_CHANGE] = (
+    "change proto3 optional flag of field `{context}.{subject}`."
+)
+_templates[FindingCategory.FIELD_BEHAVIOR_CHANGE] = (
+    "change field behavior of field `{context}.{subject}`."
+)
+_templates[FindingCategory.NEW_REQUIRED_FIELD] = (
+    "add REQUIRED field `{context}.{subject}`."
+)
+_templates[FindingCategory.FIELD_FORMAT_CHANGE] = (
+    "change field format of field `{context}.{subject}`."
+)
+_templates[FindingCategory.RESOURCE_REFERENCE_REMOVAL] = (
+    "remove resource_reference option from field `{context}.{subject}`."
+)
+_templates[FindingCategory.RESOURCE_REFERENCE_ADDITION] = (
+    "add resource_reference option to field `{context}.{subject}`."
+)
+_templates[FindingCategory.RESOURCE_REFERENCE_CHANGE] = (
+    "change resource_reference option type of field `{context}.{subject}` from `{oldtype}` to `{type}`."
```

Comment:
```suggestion
    "change resource_reference type of `{context}.{subject}` from `{oldtype}` to `{type}`."
```
maybe tighten this up a bit, as you've done in other messages? Here and elsewhere that we have "of field" to just `"of {context}.{subject}"` as well as "resource_reference option" -> "resource_reference" (which I think is a tad redundant ?) WDYT